### PR TITLE
```CNDIT-1064``: Update the Reporting service to remove the NBS page module

### PR DESCRIPTION
**Pull request by alog-enquizit**
- Update the application.yaml properties for the database parameters to be configurable. 
- Code Clean up
- Removing the NBS Page related data processing logic 
- Updating the parsing unit tests to validate the incoming Person change data.

### CODE REVIEW

File: .run/EtlDataPipelineApplication-RDS.run.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="EtlDataPipelineApplication-RDS" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
+    <envs>
+      <env name="DA_LOG_PATH" value="logs" />
+      <env name="DB_ODSE" value="nbs_odse" />
+      <env name="DB_PASSWORD" value="" />
+      <env name="DB_URL" value="cdc-nbs-legacy-rds-mssql.czya31goozkz.us-east-1.rds.amazonaws.com" />
```
Comment (upasanapattnaik-eq) [inline .run/EtlDataPipelineApplication-RDS.run.xml:7]: Would having the endpoint available in the xml raise any security concerns?

File: etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/config/KafkaConfig.java [hunk 2]
```diff
@@ -10,28 +10,15 @@
 @Configuration
 @Getter
 public class KafkaConfig {
```
Comment (upasanapattnaik-eq) [inline etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/config/KafkaConfig.java:12]: Woohoo! Glad to see this evolving.

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM

File: .run/EtlDataPipelineApplication-RDS.run.xml (hunk 1 above)
Comment (alog-enquizit) [reply .run/EtlDataPipelineApplication-RDS.run.xml:7]: Good call. I hope not. The endpoint sits behind a private VPC which means you will require a VPN connection to access.

**Pull request merged**